### ISSUE TITLE
Add Quarkus as a MicroProfile runtime to MP 3.3

### ISSUE THREAD

**Issue by pehecoro** [eclipse/microprofile-starter#303]
There is a blog post announcing 1.3 https://quarkus.io/blog/quarkus-eclipse-microprofile-3-3/, despite not having found on the implementation page https://wiki.eclipse.org/MicroProfile/Implementation

**Comment by Karm** [comment]
Hello @pehecoro, Quarkus won't be added to MP 3.3 on Starter site due to ongoing disputes.

Everything in the blog post https://quarkus.io/blog/quarkus-eclipse-microprofile-3-3/, is valid though and you can safely use it.

**Comment by bdemers** [comment]
@Karm can you provide more context around this?

**Comment by Karm** [comment]
@bdemers could you be more specific with the question, please?

**Comment by bdemers** [comment]
@Karm sorry about that!

Specifically, what are the ongoing disputes keeping Quarkus from being listed on `start.microprofile.io` under MP 3.3?

**Comment by Karm** [comment]
@bdemers Ah, that: 
Whether MicroProfile compliant runtime has also pass selected Jakarta EE TCKs, where Quarkus has its differences when it comes to CDI. Starting MicroProfile 4.1, that is no longer mandated as only MicroProfile TCKs are required: https://github.com/eclipse/microprofile/pull/215   

TL;DR:  The debate was not related to MicroProfile specification. Quarkus is passing MicroProfile TCKs. The debate was about Jakarta CDI TCK.

**Comment by bdemers** [comment]
Thanks, @Karm! very helpful!
